gc_registry/account/validation.py

```diff
@@ -23,12 +23,12 @@ def validate_account(account, read_session):
         )
 
     # All user_ids linked to the account must exist in the database
-    stmt = select(User.id).filter(User.id.in_(account.user_ids))
-    user_ids_in_db = read_session.execute(stmt).scalars().all()
-
-    if set(user_ids_in_db) != set(account.user_ids):
+    user_ids_in_db = (
+        read_session.query(User.id).filter(User.id.in_(account.user_ids)).all()
+    )
+    user_ids_in_db_set = {user_id for (user_id,) in user_ids_in_db}
+    if user_ids_in_db_set != set(account.user_ids):
```

Comment:
Something went ary in the merge from the fix you put in before by the looks, did not mean to overwrite - what this is replacing was not what was on my local branch for some reason, but the same end result is achieved